From 534c9a7ce386f1740979faa47e5f2bf25ef3c0e8 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Mon, 14 May 2018 15:29:41 +0800
Subject: [PATCH 147/151] intel-socfpga: dts: improve qspi node for rsu feature

The MTD partition used by LIBRSU must start at the beginning of
SPT0(sub-partition table 0) in flash

Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
index 73c6ce0657a7..f0408bc0031e 100644
--- a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
+++ b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
@@ -168,7 +168,7 @@
 
 			qspi_boot: partition@0 {
 				label = "Boot and fpga data";
-				reg = <0x0 0x03FE0000>;
+				reg = <0x00910000 0x036E0000>;
 			};
 
 			qspi_rootfs: partition@3FE0000 {
-- 
2.26.1

